### PR TITLE
fix: fix typo in README.md

### PR DESCRIPTION
**Pull request by cgrindel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ project.
 
 ## Toolchain setup
 
-The Apple CC toolchain in by this repository provides toolchains for
+The Apple CC toolchain in this repository provides toolchains for
 building for Apple platforms besides macOS. Since Bazel 7 this toolchain
```
Comment (cgrindel) [inline README.md:26]: I find this first sentence to be a bit confusing. Does this toolchain exist to help us build for Apple platforms when the build platform is not MacOS?

**Review by luispadron — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cgrindel) [reply README.md:26]: @keith @brentleyjones 👆

Comment (thii) [reply README.md:26]: That's right. The auto-configured CC toolchain from Bazel does not configure toolchains for iOS/watchOS/tvOS/visionOS anymore.

**Comment by cgrindel** [comment]
This should be ready to merge. Thanks!

**Pull request merged**